src/index.html

```diff
@@ -1,34 +1,76 @@
-<!doctype html>
-<html lang="en">
+<!DOCTYPE html>
+<html lang="en" class="govuk-template ">
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
-    <title>Pega - React SDK - Sample application</title>
-    <style>
-      .list {
-          font-size: large
-      }
-    </style>
-    <base href="/">
+    <meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover">
+    <meta name="theme-color" content="#0b0c0c">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="shortcut icon" sizes="16x16 32x32 48x48" href="/assets/images/favicon.ico" type="image/x-icon">
+    <link rel="mask-icon" href="/assets/images/govuk-mask-icon.svg" color="#0b0c0c">
+    <link rel="apple-touch-icon" sizes="180x180" href="/assets/images/govuk-apple-touch-icon-180x180.png">
+    <link rel="apple-touch-icon" sizes="167x167" href="/assets/images/govuk-apple-touch-icon-167x167.png">
+    <link rel="apple-touch-icon" sizes="152x152" href="/assets/images/govuk-apple-touch-icon-152x152.png">
+    <link rel="apple-touch-icon" href="/assets/images/govuk-apple-touch-icon.png">
+    <title>Claim Child Benefit - GOV.UK</title>
 
-    <link rel="icon" href="assets/img/favicon.ico">
+    <link rel="icon" href="assets/images/favicon.ico">
+    <!--TODO Remove the sdk stylev sheet link and delete from assets folder once govuk styles implemented-->
     <link rel="stylesheet" type="text/css" href="assets/css/sdkStyles.css">
     <link rel="stylesheet" type="text/css" href="assets/css/sdkCssVars.css">
-    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,300,400,500,700,900">
-    <link rel="preload" href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" as="style" type="text/css">
-    <link href="https://fonts.googleapis.com/css?family=Material+Icons&display=block" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="assets/css/govuk-frontend-4.5.0.min.css">
   </head>
-  <body>
-    <noscript>
-      <strong>The React SDK example does not work properly without JavaScript enabled. Please enable it to continue.</strong>
-    </noscript>
+  <body class="govuk-template__body ">
+    <script>
+      document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
+    </script>
 
-    <div class="flexbox">
-      <!-- React Router will put content in "outlet" -->
-      <div id="outlet" style="width: 100%;"></div>
+    <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">Skip to main content</a>
```

Comment:
We may want to split out header/footer and other display items to separate react components and reference them in the ChildBen application component.

But I think this is ok for now

Reply:
These are temporary static header and footer for MVP1, these would be removed in the final product, our app would be wrapped / embedded within HMRC header & footer